server/index.ts

```diff
@@ -23,5 +24,6 @@ app.listen(port, () => {
 
 app.use(API_VERSION + '/users', users)
 app.use(API_VERSION + '/materials', materials)
+app.use(API_VERSION + '/class', _class)
```

Comment:
This should be '/classes' since it is a collection that is being referred to. Like '/materials'.